bin/build

```diff
@@ -102,6 +103,10 @@ pushd $IMAGE_DIR
   done
 popd
 
+if [ -n "$UI_DOCKER_ONLY" ]; then
+  exit 0
+fi
+
```

Comment:
I think it might be better to introduce something like a "no-operator" parameter instead.  I've heard requests to avoid building the operator so that would solve 2 problems at once.  With that in place, then you don't need this because it will fall through the next 2 sections and exit as expected.

Reply:
See #621